instructor/client_anthropic.py

```diff
@@ -6,15 +8,19 @@
 
 @overload
 def from_anthropic(
-    client: anthropic.Anthropic | anthropic.AnthropicBedrock | anthropic.AnthropicVertex,
+    client: anthropic.Anthropic
+    | anthropic.AnthropicBedrock
```

Comment:
Does this need to be union

Reply:
You mean explicitly rewriting it as `Union[...]` as opposed to `X | Y | Z`? It would be one way of fixing it, the other way is adding `from __future__ import annotations` on top of the file, which is what I opted for because `X | Y | Z` is just such a nice syntax.

Reply:
pertect i didnt kno!